rmf_traffic/CMakeLists.txt

```diff
@@ -33,7 +33,8 @@ set(using_new_fcl true)
 set(FCL_LIBRARIES fcl)
 
 find_package(rmf_utils REQUIRED)
-find_package(Eigen3 REQUIRED)
+find_package(eigen3_cmake_module REQUIRED)
+find_package(Eigen3)
```

Comment:
Sorry to nitpick, but I think I would flip the `REQUIRED` on these.

`Eigen3` is the package that's _really_ required, while `eigen3_cmake_module` is only needed in specific environments.

Reply:
The `eigen_cmake_module` is a ROS-specific package whereas `rmf_traffic` is not meant to be ROS-specific, so I'd rather not have that kind of dependency in the ROS-agnostic build system script, even if the package manifest has it for ROS users.

> I believe in that case having QUIET could provide the user with a more confusing error

It's true, this is a tough trade-off in terms of clarity for users :thinking: